var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -301,6 +303,8 @@ class Trilinos(CMakePackage, CudaPackage):
     depends_on('py-mpi4py', when='+mpi+python', type=('build', 'run'))
     depends_on('py-numpy', when='+python', type=('build', 'run'))
     depends_on('python', when='+python')
+    depends_on('python', when='+ifpack', type='build')
+    depends_on('python', when='+ifpack2', type='build')
```

Comment:
It looks like 13.0.1 don't require python, so:
```suggestion
    depends_on('python', when='@13.2: +ifpack', type='build')
    depends_on('python', when='@13.2: +ifpack2', type='build')
```

Reply:
Looks like the edit for `+hypre` didn't post in time for you to see sorry :(

Reply:
thanks, pushed!